hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/utils/SimpleStripedTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package org.apache.hadoop.hdds.utils;
+
+import com.google.common.util.concurrent.Striped;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReadWriteLock;
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+/**
+ * Test cases for {@link SimpleStriped}.
+ */
+public class SimpleStripedTest {
```

Comment:
Should be `TestSimpleStriped` -- currently it is not executed in CI.

Reply:
Thanks for the review, @adoroszlai. Updated.

Reply:
Verified that it's executed in CI now.
```
[INFO] Running org.apache.hadoop.hdds.utils.TestSimpleStriped
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.049 s - in org.apache.hadoop.hdds.utils.TestSimpleStriped
```